PROJECT

```diff
@@ -6,4 +6,14 @@ plugins:
   scorecard.sdk.operatorframework.io/v2: {}
 projectName: ovs-operator
 repo: github.com/openstack-k8s-operators/ovs-operator
+resources:
+- api:
+    crdVersion: v1
+    namespaced: true
+  controller: true
+  domain: openstack.org
+  group: ovs
+  kind: OVS
+  path: github.com/openstack-k8s-operators/ovs-operator/api/v1alpha1
+  version: v1alpha1
```

Comment:
We've been starting the operator APIs at version `v1beta1`, so it's something you might consider doing here as well instead of `v1alpha1`

Reply:
Done